README.md

```diff
@@ -65,7 +65,11 @@ xcode-select --install
 If you are running Ubuntu 20.04+ you can use the commands to install the dependencies:
 ```
 sudo apt update && sudo apt install make jq zip gcc python3-pip
+sudo rm -rf /usr/local/go && sudo wget https://go.dev/dl/go1.16.11.linux-amd64.tar.gz && sudo tar -C /usr/local -xzf go1.16.11.linux-amd64.tar.gz
```

Comment:
It looks like there is a 1.16 package for 20.04, though it's a little older.  Could you update this to suggest installing that instead? 
https://packages.ubuntu.com/focal-updates/golang-1.16

Replacing the files will cause the currently installed package to fail to upgrade or uninstall.

Reply:
Weird: the focal-updates repository is listed in my software sources, and yet trying to update to it fails.

This is probably Mint-specific in some way and so not worth mentioning.